test/unit/base/base-command.test.ts

```
@@ -0,0 +1,289 @@
+import fs from "node:fs";
+import { AblyBaseCommand } from "../../../src/base-command.js";
+import { ConfigManager } from "../../../src/services/config-manager.js";
+import { InteractiveHelper } from "../../../src/services/interactive-helper.js";
+import { BaseFlags } from "../../../src/types/cli.js";
+import { Config } from "@oclif/core";
+
+// Create a testable implementation of the abstract AblyBaseCommand
+class TestCommand extends AblyBaseCommand {
+  // Expose protected methods for testing
+  public testCheckWebCliRestrictions(): void {
+    return this.checkWebCliRestrictions();
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)